.github/workflows/images.yaml

```diff
@@ -1,6 +1,8 @@
 name: Images
 
 on:
+  pull_request:
```

Comment:
It's out of the scope of this pull request, but I just noticed that we missed adding `on` for tags(We might need it as we push images only on tags.